documentation/modules/oauth/proc-oauth-authorization-broker-config.adoc

```diff
@@ -24,7 +24,6 @@ NOTE: xref:property-simple-authorization-superusers-reference[Super users] alway
 * Strimzi must be configured to use OAuth 2.0 with Keycloak for xref:assembly-oauth-authentication_str[token-based authentication].
 You use the same Keycloak server endpoint when you set up authorization.
 * OAuth 2.0 authentication must be configured with the `maxSecondsWithoutReauthentication` option to enable re-authentication.
-* You need to understand how to manage policies and permissions for Keycloak Authorization Services, as described in the {keycloak-server-doc}.
```

Comment:
Are we linking to the new sections instead?

Reply:
The new thing comes right after this chapter, and is not required for configuring the authorization in Strimzi, but for configuring the authorization rules in Keycloak. It's not a precondition / requirement for the steps that follow.

Reply:
Okay